src/lib/nftClaimer/utils.ts

```diff
@@ -132,3 +135,18 @@ export function validateAddresses(addresses: Record<string, string>) {
 
   return true;
 }
+
+export async function snapshotFee() {
+  try {
+    const provider = snapshot.utils.getProvider(NFT_CLAIMER_NETWORK);
+    const contract = new Contract(
+      DEPLOY_CONTRACT,
+      ['function snapshotFee() public view returns (uint8)'],
+      provider
+    );
+    return contract.snapshotFee();
```

Comment:
I mean like this, the provider and contract won't change unless we change env
```suggestion
const provider = snapshot.utils.getProvider(NFT_CLAIMER_NETWORK);
const contract = new Contract(
  DEPLOY_CONTRACT,
  ['function snapshotFee() public view returns (uint8)'],
  provider
);
export async function snapshotFee() {
  try {
    return contract.snapshotFee();
```

Reply:
Those variables are not used outside this function. Keeping them inside will save some memory, as they'll be instantiated only when needed.

Reply:
oh i see, thank you for explaining